source/Jobbr.ComponentModel.Registration/IJobbrBuilder.cs

```diff
@@ -7,5 +7,9 @@ public interface IJobbrBuilder
         void Register<T>(Type type);
 
         void Add<T>(object instance);
+
+        void AppendInstanceToCollection<T>(object instance);
+
+        void AppendTypeToCollection<T>(Type type);

```

Comment:
I suggest a more consistent naming convention that also doesn't leak as much implementation details.

```
        void AddForCollection<T>(Type type);

        void RegisterForCollection<T>(object instance);
```